testsuite/WIKI.md

```diff
@@ -0,0 +1,198 @@
+```
+
+## Alternatives
+You can install `MDAnalysisTests` and then run the tests anywhere. Extra functionality, afforded by our nose plugins, is added only if the tests are run through the `mda_nosetests` script, or by directly invoking `MDAnalysis.tests.test()` (which is what the 3-line `mda_nosetests` script does under the hood).
+
+You can run all tests in serial from the python interpreter like this:
+```python
+from MDAnalysis.tests import test
+test()
+```
+
+Fore more details see below.
+
+## Examples ##
+Examples for output in various modes. For debugging, verbose mode is more useful as one can identify failing tests while they are running.
+
+### Serial testing ###
+For example, a successful test might look like the following
+```
+>>> MDAnalysis.tests.test()
+......S...S............................................................................................................................................................K.KK...........................................................................................................................................................................K...............................................................................................................................................................................................................K..............................................................K............................................................................................................................................................................................................................................................................................................................................................................K......K....................K.....................K...................K....................K...........................K...................K...................K.............................................................K...................K........................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................................
+============ 5497 passed, 31 skipped, 1 xfailed in 1056.77 seconds =============
+```
+
+### Coverage ###
+We test code coverage of the unit tests with the  [coverage](http://nedbatchelder.com/code/modules/rees-coverage.html) plugin of nose. Currently, this is done automatically as part of the Travis job on Python 2.7 and is viewable on **[coveralls](https://coveralls.io/r/MDAnalysis/mdanalysis?branch=develop)**.
```

Comment:
If we mention coverage then this should be installed with `setup.py` (it is not currently installed).